dts/bindings/ipc/zephyr,ipc-icmsg.yaml

```diff
@@ -21,6 +21,10 @@ properties:
     required: true
     type: phandle
 
+  cache-line-sz:
```

Comment:
I will check if this can be done in a nice way.

Reply:
If both does not implement cache, you omit this property. If any side use cache on memory region defined, you define `cache-line-sz` as the biggest cache line size from both sides.

Reply:
In this case, I suggest to keep this property, but document properly how to use it when cache lines differ on both sides of communication

Reply:
I will rename that property since it already cased a confusion. In fact it does not define `cache_line_sz` but cache alignment for communication channel, which could be local or remote. Depends on which value is bigger.

Reply:
@hubertmis Please revisit. I hope it's better described now.